junit-vintage-engine/src/main/java/org/junit/vintage/engine/VintageTestEngine.java

```diff
@@ -36,6 +48,10 @@
 @API(status = INTERNAL, since = "4.12")
 public final class VintageTestEngine implements TestEngine {
 
+	private static final String PARALLEL_CONFIG_PREFIX = "junit.jupiter.execution.parallel";
```

Comment:
Should be `junit.vintage.execution.parallel`.

Reply:
Will the Maven Surefire and Failsafe plugins set these properties? I was not sure where to find the documentation on this service boundary?

Reply:
@marcphilipp, I made this change and documented all of the parameters in the user guide as well. I re-tested with the Maven Surefire plugin as well and it appears to behave as expected.

Reply:
Surefire doesn't set these properties but users can set them in their Surefire configuration.